src/root/components/connected/alerts-table.js

```diff
@@ -4,17 +4,18 @@ import { PropTypes as T } from 'prop-types';
 import { Link } from 'react-router-dom';
 import { DateTime } from 'luxon';
 
-import { environment } from '../../config';
-import { getSurgeAlerts } from '../../actions';
-import { get, dateOptions, datesAgo, isLoggedIn } from '../../utils/utils';
-import { nope, privateSurgeAlert, recentInterval } from '../../utils/format';
-
-import ExportButton from '../export-button-container';
-import { SFPComponent } from '../../utils/extendables';
-import DisplayTable, { FilterHeader } from '../display-table';
-import BlockLoading from '../block-loading';
-import Fold from '../fold';
-import Expandable from '../expandable';
+import { environment } from '#root/config';
+import { getSurgeAlerts } from '#actions';
+import { get, dateOptions, datesAgo, isLoggedIn } from '#utils/utils';
+import { nope, privateSurgeAlert, recentInterval } from '#utils/format';
+
+// FIXME: imports from the /components/ could be a 1 liner?
```

Comment:
@frozenhelium some of these could be 1 liners in most files?

Reply:
How do you mean?

Reply:
Something like (could be wrong, didn't have the time yet to mess with this much):

```javascript
import ExportButton, DisplayTable, FilterHeader, ...etc from '#components';
```

`FilterHeader` might not work like this since it's not a default export, but somehow like this.

Reply:
That's true, we can add a `/components/index.js` file and re-export every components from there.  But I think, it's better to import components separately. Lets see if we can do that later.

Reply:
I don't know if that's actually necessary, like default exports should work simply like this as far as I know. I'm just not 100% sure. Have to test it later.